rasa_nlu/evaluate.py

```diff
@@ -81,19 +81,22 @@ def plot_confusion_matrix(cm, classes,
     plt.xlabel('Predicted label')
 
 
-def log_evaluation_table(test_y, preds):  # pragma: no cover
+def get_evaluation_table(test_y, preds, log=True):  # pragma: no cover
```

Comment:
lets split this into two:
`get_evaluation_table(...)` like this, but without the log
`log_evaluation_table(...)` that calls `get_evaluation_table(...)` and logs the result

The functions have two different intentions and I'd like to avoid parameters that select a functions logic based on their truth value.